packages/manager/modules/anthos/src/dashboard/host/restart/controller.js

```diff
@@ -0,0 +1,23 @@
+export default class {
+  /* @ngInject */
+  constructor($translate, AnthosTenantsService) {
+    this.$translate = $translate;
+    this.AnthosTenantsService = AnthosTenantsService;
+  }
+
+  restartHost() {
+    this.AnthosTenantsService.restartHost(this.serviceName, this.host.id)
+      .then((hostData) => {
+        this.host.update(hostData);
+        this.goBack();
```

Comment:
no success message?

Reply:
A success message is not required here, since the state of the host is directly updated